### PR TITLE
(doc) Use parameter and argument appropriately in Options

### PR DESCRIPTION
**Pull request by PoorlyDefinedBehaviour**
Part of https://github.com/ocaml/ocaml.org/issues/1911

### CODE REVIEW

File: data/tutorials/language/3ds_00_options.md [hunk 1]
```diff
@@ -157,14 +157,14 @@ val g : 'a list -> 'a option = <fun>
 ```
 
 The types of `f` and `g` are reversed. Functions `Option.fold` and `option_unfold_try` also work in a reversed manner
-* `Option.fold` takes an option parameter to build something
-* `option_unfold_try` returns an option result from some parameter
+* `Option.fold` takes an option argument to build something
```
Comment (cuihtlauac) [inline data/tutorials/language/3ds_00_options.md]:
```suggestion
* `Option.fold` has as an option argument
```

File: data/tutorials/language/3ds_00_options.md [hunk 2]
```diff
@@ -157,14 +157,14 @@ val g : 'a list -> 'a option = <fun>
...
+* `option_unfold_try` returns an option result from some argument
```
Comment (cuihtlauac) [inline data/tutorials/language/3ds_00_options.md]:
```suggestion
* `option_unfold_try` returns an option result

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

File: data/tutorials/language/3ds_00_options.md [hunk 3]
```diff
@@ -157,14 +157,14 @@ val g : 'a list -> 'a option = <fun>
...
-* `option_unfold_try` returns an option result from some parameter
+* `Option.fold` has as an option argument
+* `option_unfold_try` returns an option result
 
 The standard library does not contain any option unfolding function.
 
 ### Bind an Option
 
-The `bind` function of type `'a option -> ('a -> 'b option) -> 'b option` works a bit like `map`. But whilst `map` expects a function-as-parameter `f` that returns an unwrapped value of type `b`, `bind` expects an `f` that returns a value already wrapped in an option `'b option`.
+The `bind` function of type `'a option -> ('a -> 'b option) -> 'b option` works a bit like `map`. But whilst `map` expects a function-as-argument `f` that returns an unwrapped value of type `b`, `bind` expects an `f` that returns a value already wrapped in an option `'b option`.
```
Comment (cuihtlauac) [inline data/tutorials/language/3ds_00_options.md]:
```suggestion
The `bind` function of type `'a option -> ('a -> 'b option) -> 'b option` works a bit like `map`. But whilst `map` expects a function parameter `f` that returns an unwrapped value of type `b`, `bind` expects an `f` that returns a value already wrapped in an option `'b option`.
```

File: data/tutorials/language/3ds_00_options.md [hunk 4]
```diff
@@ -175,7 +175,7 @@ Here, we display the type of `Option.map`, with parameters flipped and show a po
 val bind : 'a option -> ('a -> 'b option) -> 'b option = <fun>
 ```
 
-Observe that the types are the same, except for the codomain of the function-as-parameter.
+Observe that the types are the same, except for the codomain of the function-as-argument.
```
Comment (cuihtlauac) [inline data/tutorials/language/3ds_00_options.md]:
```suggestion
Observe that the types are the same, except for the codomain of the function parameter.
```

File: data/tutorials/language/3ds_00_options.md [hunk 5]
```diff
@@ -157,14 +157,14 @@ val g : 'a list -> 'a option = <fun>
 ```
 
 The types of `f` and `g` are reversed. Functions `Option.fold` and `option_unfold_try` also work in a reversed manner
-* `Option.fold` takes an option parameter to build something
-* `option_unfold_try` returns an option result from some parameter
+* `Option.fold` has as an option argument
```
Comment (cuihtlauac) [inline data/tutorials/language/3ds_00_options.md]:
```suggestion
* `Option.fold` has as an option parameter
```

**Review by cuihtlauac — DISMISSED** [review]

**Review by sabine — APPROVED** [review]
Thanks @PoorlyDefinedBehaviour! I applied the changes from @cuihtlauac, so merging now. :sparkles: :+1:

**Pull request merged**